autometa/common/external/samtools.py

```diff
@@ -61,8 +89,8 @@ def main(args):
         format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p')
     parser = argparse.ArgumentParser()
-    parser.add_argument('--bam')
-    parser.add_argument('--sam')
-    parser.add_argument('--seqs')
+    parser.add_argument('--sam', help='</path/to/alignment.sam>')
+    parser.add_argument('--out', help='</path/to/output/file')
```

Comment:
Similarly, you could place the `.bam` extension on the output file like so, `</path/to/output/file.bam>`